include/llbuild/Basic/ExecutionQueue.h

```diff
@@ -26,6 +26,9 @@ namespace llbuild {
     
     class ExecutionQueueDelegate;
 
+    // Default shell system path
+    const std::string ShellPath = "/bin/sh";
```

Comment:
No particular reason to define this here, other than this header is already directly included by three of the four files where this variable's needed. I can move it elsewhere if preferred.

Reply:
Seems ok to start. Maybe DefaultShellPath?

Reply:
Done.